Cargo.lock

```diff
@@ -5369,6 +5370,27 @@ dependencies = [
  "winnow",
 ]
 
+[[package]]
+name = "tower"
```

Comment:
Do we mind this? 🤔

Reply:
https://crates.io/crates/tower

I don't think pulling it in is a big deal, just thought I'd ask for a second opinion...